src/ffmpeg.ts

```diff
@@ -410,6 +410,7 @@ const completionSpec: Fig.Spec = {
     {
       name: "-intra",
       description: "Deprecated use -g 1",
+      deprecated: true,
```

Comment:
@ryanccn We can provide more info for deprecated: https://fig.io/docs/reference/option#deprecated.
```suggestion
      deprecated: {
        insertValue: "-g 1",
        description: "This option has been deprecated",
      },
```

Reply:
Hmm, that would be a bit difficult

Reply:
I'll look at it. How about you create an issue in [ryanccn/ffmpeg-fig](https://github.com/ryanccn/ffmpeg-fig)? Have all the issues about the generator in one place